forms-flow-bpm/src/main/resources/application.yaml

```diff
@@ -17,6 +17,9 @@ server:
         max-age: 1800
 
 formsflow.ai:
+  enableCustomSubmission: ${CUSTOM_SUBMISSION_ENABLED}
```

Comment:
you can create a subproperty called forms and place the new properties under it. Make more sense